docs/source/starter/installation.rst

```diff
@@ -0,0 +1,55 @@
+.. _installation:
+
+############
+Installation
+############
+
+PyTorch Lightning is maintained and tested on Pytorch 3.7.0+ and PyTorch 1.7.0+.
```

Comment:
This should be a more exhaustive matrix, similar to the readme.

Reply:
I can improve it a little more, but adding that exhaustive matrix might be an overkill. I don't think users might be interested in what kind of tests we cover in the docs. Plus we need to make sure that we update this particular doc every time a small change is made in the ci config. As far as I can tell, the coverage on README is a little outdated too.

Reply:
If this is already in the README, I would not copy it here.

We will forget about this eventually. My suggestion is to remove it or link to the README.

Reply:
linked the README.